From 662bae245446354f0e0852d609258dcb7c2d1e13 Mon Sep 17 00:00:00 2001
From: Matthew DeGarmo <46715299+TechDufus@users.noreply.github.com>
Date: Thu, 18 Dec 2025 20:21:13 -0600
Subject: [PATCH 0455/5749] feat(lsp): add bash-language-server to builtin
 servers (#112)

---
 src/tools/lsp/constants.ts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/tools/lsp/constants.ts b/src/tools/lsp/constants.ts
index a585e638..fe921192 100644
--- a/src/tools/lsp/constants.ts
+++ b/src/tools/lsp/constants.ts
@@ -109,6 +109,10 @@ export const BUILTIN_SERVERS: Record<string, Omit<LSPServerConfig, "id">> = {
     command: ["astro-ls", "--stdio"],
     extensions: [".astro"],
   },
+  "bash-ls": {
+    command: ["bash-language-server", "start"],
+    extensions: [".sh", ".bash", ".zsh", ".ksh"],
+  },
   jdtls: {
     command: ["jdtls"],
     extensions: [".java"],
-- 
2.39.2

