From a24f1e905e060ff9daec17f6b6d6018bdc06a2a6 Mon Sep 17 00:00:00 2001
From: justsisyphus <justsisyphus@users.noreply.github.com>
Date: Sun, 1 Feb 2026 14:43:30 +0900
Subject: [PATCH 1610/5749] chore: fix bun-build gitignore pattern to catch all
 variants

---
 .gitignore | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.gitignore b/.gitignore
index 5c4708d6..2f4f5ebf 100644
--- a/.gitignore
+++ b/.gitignore
@@ -33,4 +33,4 @@ yarn.lock
 test-injection/
 notepad.md
 oauth-success.html
-.188e87dbff6e7fd9-00000000.bun-build
+*.bun-build
-- 
2.39.2

